package.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>fastrtps</name>
+  <version>2.0.0</version>
+  <description>
+     C++ implementation of the DDS standard of the OMG.
+  </description>
+  <maintainer email="miguelcompany@eprosima.com ">Miguel Company</maintainer>
+  <maintainer email="eduardoponz@eprosima.com ">Eduardo Ponz</maintainer>
+  <license file="LICENSE">Apache 2.0</license>
+
+  <url type="website">https://www.eprosima.com/</url>
+  <url type="bugtracker">https://github.com/eProsima/Fast-DDS/issues</url>
+  <url type="repository">https://github.com/eProsima/Fast-DDS</url>
+
+  <buildtool_depend>cmake</buildtool_depend>
+
+  <depend>fastcdr</depend>
+
+  <build_depend>foonathan_memory</build_depend>
+  <build_depend>Asio</build_depend>
+  <build_depend>TinyXML2</build_depend>
+
+  <!--
+   Only with SECURITY
+  <build_depend>OpenSSL</build_depend>
+
+   Only with ANDROID
+  <build_depend>android-ifaddrs</build_depend>
+  -->
```

Comment:
Why is this commented out?

Reply:
These are dependencies just used with specific building args.
They are left commented in case in the future they are needed, and to help any user that needs to find project dependencies.